src/daffodilDebugger.ts

```diff
@@ -189,7 +189,14 @@ export async function getDebugger(
       // Get daffodilDebugger class paths to be added to the debugger
       let daffodilDebugClasspath = ''
 
+      //check if each classpath still exists
       if (config.daffodilDebugClasspath) {
+        config.daffodilDebugClasspath.split(':').forEach((classpath) => {
+          if (!fs.existsSync(classpath)) {
+            throw new Error(`File or directory: ${classpath} doesn't exist`)
+          }
+        })
+
         daffodilDebugClasspath = config.daffodilDebugClasspath.includes(
```

Comment:
What does this line do? 

Is this setting a default value of the workspace dir as the only content of daffodilDebugClasspath? If so I would not expect this as the default.